docs/blog/news-and-updates/2023-06-13-update.mdx

```diff
@@ -0,0 +1,106 @@
+# Developer Weekly - June 13, 2023
+
+Hello devs and welcome to this week’s issue of Developer Weekly! This week, we’re going to break down our latest release of dfx (0.14.1) by diving into the new features that shipped with this new version, as well as talk about some new features under the hood of Motoko. 
+
+Let’s dive in!
+
+## dfx 0.14.1
+The latest release of dfx, 0.14.1, primarily addresses issues that were experienced in the previous release, 0.14.0; one of these issues was a bug within the asset canister feature that affected single-page applications. For this reason, we strongly recommend that users upgrade to 0.14.1.
+
+In addition to the important patch for the asset canister functionality, dfx shipped with new features to enhanced developer experiences, such as:
+
+### Feature: gzip option in dfx.json: 
+
+dfx can now use gzip on the wasm module as a final step when building canisters. This feature is disabled by default, but can be enabled in the `dfx.json` file by using the following configuration:
+
+```
+{
+  "canisters" : {
+    "app" : {
+      "gzip" : true
+    }
+  }
+}
+```
+
+If you want to specify the `.wasm.gz` file directly for custom canisters, this functionality is still supported. Metadata, optimization, or shrink options that are set in the `dfx.json` file will be applied after the custom `.wasm.gz` file is decompressed, modified, then compressed back into `wasm.gz` at the end of the canister build process. 
+
+### Feature: dfx deps subcommands
```

Comment:
I would hold off on announcing this until we @lwshang has finished the invoice canister integration. Otherwise, devs will be using an empty feature. Goal is to make a splashy release!